woof-code/rootfs-skeleton/usr/sbin/bootmanager

```diff
@@ -327,7 +327,7 @@ XML_LABEL=$(gettext 'Run pupswap')
 XML_ACTION='pupswap &'
 SWAPFILE_XML="$(generic_xml)" #--
 
-if [ $PUPMODE -eq 2 ];then #PUPMODE=2, full hd install, will not have this.
+if [ $PUPMODE -eq 2 -o $PUPMODE -eq 6 ];then #PUPMODE=2, full hd install, will not have this.
```

Comment:
TODO: this is the wrong check. The UI is misleading and says SFS loading is unsupported.